Mihai-Blaga/COMP30022
# Tasks/AA-36 Cancel Meeting

This branch contains code from AA-85. AA-85 should be merged before merging this in. You can test and merge that branch in if you want.

## Changes
- Added delete meeting button in fragment xml
- Created deleteMeeting function in CalenderUtil

## Tests
- Click on any meeting on upcoming meetings
- Press delete button
- Verify that event has been deleted in CRM app and Google Calendar
